src/index.json

```diff
@@ -4712,7 +4712,8 @@
                         "Programming Language :: Python :: 3.5",
                         "Programming Language :: Python :: 3.6",
                         "Programming Language :: Python :: 3.7",
-                        "Programming Language :: Python :: 3.8"
+                        "Programming Language :: Python :: 3.8",
+                        "Programming Language :: Python :: 3.9"
```

Comment:
Did we started supporting Python 3.9 for ml cli?

Reply:
Thanks for catching it. Have made the necessary update.